### ISSUE TITLE
App Manager not working

### ISSUE THREAD

**Issue by nikilase** [LedgerHQ/ledger-live-desktop#943]
#### Ledger Live Version and Operating System

- Ledger Live 1.0.0
- Platform: Windows 10

#### Actual behavior
After adding an Account (currency) to Ledger Live and then going to the Manager Tab I get stuck on "Navigate to the dashboard on your device" although I am on the dashboard on my device.

#### Steps to reproduce the behavior
-open currency app on Ledger
-add currency to the Portfolio in Ledger Live
-close currency app on Ledger
-navigate to Manager in Ledger Live

**Comment by michi6383** [comment]
can confirm that.

**Comment by elkimek** [comment]
Same here.

**Comment by Deozaan** [comment]
I had the same issue (as well as #949). Restarting Ledger Live resolved both issues for me.

**Comment by nyanloutre** [comment]
I have this too on Linux

**Comment by gre** [comment]
this should be fixed in 1.0.2. either wait we switch the prerelease to a release or manually install the current prerelease :)

**Comment by gre** [comment]
we'll reopen if it's not fixed

**Comment by nyanloutre** [comment]
Still blocking on "Allow Ledger Manager on your device" without seeing any prompt on screen

**Comment by prometx** [comment]
Same issue as nyanloutre, ubuntu 18.04...

**Comment by ryanpaulfyfe** [comment]
Still see this issue in V 1.0.3 on ubuntu 18.04

**Comment by gre** [comment]
can you raise a new issue? the original issue is not the same problem (was stuck at Dashboard step, which was definitely a bug that was fixed on Windows)

**Comment by ryanpaulfyfe** [comment]
Sorry about that, Done. https://github.com/LedgerHQ/ledger-live-desktop/issues/1220

**Comment by PietroPasotti** [comment]
same here on 1.0.3: "Navigate to the dashboard on your device"
also the ledger manager chrome app won't work, is stuck on "connect your device".

**Comment by gre** [comment]
@PietroPasotti please check these guide to see if it helps: https://support.ledgerwallet.com/hc/en-us/articles/115005165269-Fix-connection-issues

**Comment by PietroPasotti** [comment]
This fixed the desktop app access (for now), but not the chrom(ium) app.
Thanks!

**Comment by fazlearefin** [comment]
Had the same issue on Ubuntu 18.04. Solved the issue by running as root

```console
sudo ./ledger-live-desktop-1.0.5-linux-x86_64.AppImage
```

**Comment by meriadec** [comment]
@fazlearefin not sure this is a good workaround to give admin privilege to any app. Can you verify you have correctly set `udev` rules (see https://github.com/LedgerHQ/ledger-live-desktop/issues/943#issuecomment-407644512)

**Comment by fazlearefin** [comment]
@meriadec Ok, I had not set the udev rules. The Ledger Live installation instructions in https://support.ledgerwallet.com/hc/en-us/articles/360006395553 should clearly mention adding the udev rules before starting Ledger Live.

**Comment by meriadec** [comment]
definitely. it should improve with time. see https://github.com/LedgerHQ/ledger-live-desktop/issues/1292

**Comment by jeliasson** [comment]
Same problem on version 1.3.4

**Comment by rmokros** [comment]
still a problem ubuntu 18.01  ledger live version 1.4.1

**Comment by inad9300** [comment]
Same issue over here with Ubuntu LTS...

**Comment by prometx** [comment]
After tinkering with this for some time, I solved this on Ubuntu 18.04.1 LTS, by moving the usb cable from port to port. It turns out, some USB ports on many motherboards, aren't shielded correctly, and that trying all available port may help in getting an "clean connection" to the ledger. 

That is my theory at any rate, I had been trying via a USB Hub; unplugged from there, tried several builtin ports of the machine and, voila...🤓

**Comment by raspatan** [comment]
running with sudo "solved" it for me. Ubuntu 16.04.
